app/serializers/disability_claim_base_serializer.rb

```diff
@@ -30,4 +27,44 @@ def open
   def waiver_submitted
     object.data['waiver5103Submitted']
   end
+
+  def development_letter_sent
+    bool_from_yes_no 'developmentLetterSent'
+  end
+
+  def decision_letter_sent
+    bool_from_yes_no 'decisionNotificationSent'
+  end
+
+  protected
+
+  def with_object_data(*keys)
+    val = object.data.dig(*keys)
+    yield val if val.present?
+  end
+
+  def list_objects_with_key(parents, sub_keys)
+    parent = object.data.dig(*parents)
+    parent = [] if parent.blank?
+    parent.each do |obj|
+      val = obj.dig(*sub_keys)
+      obj if val.present?
+    end.compact
+  end
+
+  def bool_from_yes_no(*keys)
+    with_object_data(*keys) do |s|
+      case s.downcase
+      when 'yes' then true
+      when 'no' then false
+        # TODO: Log weird input
```

Comment:
Can we do this TODO now? I would just add the `else` case and log the input with a helpful message, and either return `nil` or `false`